registration/registration-services/src/main/java/io/mosip/registration/mdm/service/impl/MosipDeviceSpecificationHelper.java

```diff
@@ -66,7 +66,7 @@ public String getPayLoad(String data) throws RegBaseCheckedException {
 
 	public MdmDeviceInfo getDeviceInfoDecoded(String deviceInfo) {
 		try {
-			validateJWTResponse(deviceInfo);
+			validateJWTResponse(deviceInfo, "DEVICE");
```

Comment:
please define a constant and pick from there. or can we not pick this from properties?

Reply:
changed to use properties